front/lib/resources/agent_message_feedback_resource.ts

```diff
@@ -346,15 +347,14 @@ export class AgentMessageFeedbackResource extends BaseResource<AgentMessageFeedb
           as: "agentMessage",
           attributes: [
             "id",
-            "sId",
             "agentConfigurationId",
             "agentConfigurationVersion",
           ],
         },
       ],
     });
 
-    if (!message || !message.agentMessageId) {
+    if (!message || !message.agentMessage) {
```

Comment:
Please make sure to test your code always. Really. This was clearly untested right?